web/client/plugins/FeaturedMaps.jsx

```diff
@@ -66,20 +66,32 @@ class FeaturedMaps extends React.Component {
                 title={<h3><Message msgId="manager.featuredMaps" /></h3>}
                 maps={items}
                 colProps={this.props.colProps}
-                viewerUrl={(res) => {
-                    if (res.category && res.category.name === "DASHBOARD") {
-                        return this.context.router.history.push(`/dashboard/${res.id}`);
-                    }
-                    if (res.category && res.category.name === "GEOSTORY") {
-                        return this.context.router.history.push(`/geostory/${res.id}`);
-                    }
-                    return this.context.router.history.push("/viewer/" + this.props.mapType + "/" + res.id);
-                }}
+                viewerUrl={(res) => this.context.router.history.push('/' + this.makeShareUrl(res).url)}
+                shareUrl={this.makeShareUrl}
```

Comment:
hi, if this is a function i wouldCall it
```suggestion
                getShareUrl={this.makeShareUrl}
```
please change also other references to shareUrl